src/api/config/routes.rb

```diff
@@ -281,7 +281,7 @@ def self.public_or_about_path?(request)
       # TODO: this should be POST (and the link AJAX)
       get 'project/toggle_watch/:project' => :toggle_watch, constraints: cons, as: 'project_toggle_watch'
       get 'project/meta/:project' => :meta, constraints: cons, as: 'project_meta'
-      post 'project/save_meta/:project' => :save_meta, constraints: cons
+      post 'project/save_meta/:project' => :save_meta, constraints: cons, as: :project_meta_config
```

Comment:
why not `project_save_meta`? :confused: 

I am not sure I like the `as` in routes. They are not the default anyway, so you have to look them up. :thinking: